DOCUMENTATION/Getting_Started_CLI.md

```diff
@@ -0,0 +1,310 @@
+# Getting Started with the CLI
+
+Cauldron can be used as either through its Command Line Interface (CLI) or with the Cauldron desktop application. This document explains using Cauldron through its CLI. To install the Cauldron desktop app you can instructions here: http://www.unnotebook.com/install.
```

Comment:
Missing word here: *To install the Cauldron desktop app you can **?** instructions ...*

Reply:
Corrected the sentence to: <em>To install the Cauldron desktop app you may view the instructions here: </em>